R/derive_param_response.R

```diff
@@ -62,6 +62,14 @@
 #'
 #'   The values must be symbols, character strings, numeric values or `NA`.
 #'
+#' @param aval_fun Function to map character analysis value (`AVALC`) to numeric
+#'   analysis value (`AVAL`)
+#'
+#'   The (first) argument of the function must expect a character vector and the
+#'   function must return a numeric vector.
+#'
+#'   *Default:* `yn_to_numeric` (see `admiral::yn_to_numeric()` for details)
```

Comment:
There was a decision to remove all defaults, so probably better to remove this here too.

Reply:
That's right. However, it provides a link to the default function. Thus I would keep it.

Reply:
I removed all defaults from all other functions in accordance with the response from @rossfarrugia and @thomas-neitmann to this issue: https://github.com/pharmaverse/admiralonco/issues/130

So this is the only function with a default now, in admiralonco.

Reply:
i'd remove it for now for consistency with Stephen's PR we merged that deleted the rest of these. if we get feedback that people are struggling to find this then we could always add a note without using the `*Default:*` convention, but users could always just use `??yn_to_numeric()` if stuck

Reply:
Removed the *Default.....  from the header